README.md

```diff
@@ -9,14 +9,16 @@ A plugin that allows you to use React as a templating language for Eleventy. Thi
 
 This plugin requires `react`, `react-dom`, `react-helmet`, `@babel/core`, and `babel-loader` as peer dependencies to allow you to have control over which version of these packages you're using.
 
+The `.eleventy.js` snippet below also requires `@babel/preset-react` and `@babel/preset-env` be installed.
```

Comment:
As I continue to think about this, I wonder if we should just make these peer deps for now? The config will almost always require `@babel/preset-react`, and `@babel/preset-env` is the prescribed preset for ensuring your code is compiled for the correct targets. Thoughts?

Reply:
Just wanted to +1 my support for “Sane Defaults”.

Reply:
I'm seriously considering just setting up all the Babel stuff internally under the hood and providing an API that allows users to customize that config - will have to figure out how best to pull Babel presets and plugins from both the package and from the root project, but I'm sure it can be done!

Reply:
@dpfavand Would you like to update this to add the two packages to the peer deps list? I can merge this then :)

Reply:
Sure - they are now in the peer deps. 

I've also modified the readme copy to just list them as peer deps that must be installed.